### DISCUSSION TITLE
허태영 1회차 블로그 피드백 요청

### DISCUSSION THREAD

**Discussion by taeyoung0001** [Orm-camp/blog-feedback#5] [동글동글 2기 피드백]
https://blog.naver.com/tae_022/223572195441

피드백 요청 부탁드립니다 ㅎㅎ
달게 받겠습니다

**Comment by juuyeoon** [comment]
태영님 화이팅!! 두근두근하면서 글 기다리고 있을게요 ㅎㅎ
(사실 저 전에 쓰다만 글도 올려서 피드백 받고 그랬어요...ㅋㅋㅋ 너무 완벽하지 않아도 되니 마음 편하게 올려보세요!)

**Comment by hyyun1113** [comment]
태영님! 글 잘 읽었습니다! 역시 프로젝트 발표할때 달변가임을 눈치챘지만 그게 블로그에서도 느껴지는군요..! 엄청 술술 재미있게 읽었습니다.
다만 프로젝트시기는 언제인지등 객관적 지표도 추가하면 어떨까하는 생각이 들었습니다.  
재밌는 블로그 글, 다음에도 기대할게요!

**Comment by dayoungh** [comment]
> 수업할 때는 쉬워 보였던 문법들이 제가 직접 만들어 보려니깐 너무 어려웠었죠,,
> 가장 큰 맨붕이 올 때는 알 수 없는 빨간 줄들이 생길 때였습니다..

이 부분 너무 공감갑니다😂 너무 신경쓰이고 스트레스 받는 빨간 줄..!!! 
체계화 되어 있으면서도 편하게 쓰신 글이라 재밌게 읽을 수 있었습니다! 은솔님도 그렇고 태영님도 그렇고 저렇게 이미지 첨부해주신 게 너무 좋은 아이디어 같아요,, 배우고 갑니다👏👏👏

**Comment by ggaong** [comment]
프로젝트 중간 중간 웃음 주시던 태영님 덕분에 안지치고 잘 끝낼 수 있지 않았나 싶습니다😄
쓰신 내용 중에 "오류로 한참 고생했는데 별 거 아니었던 걸 알면 느껴지는 허탈함" 이게 너무 공감됩니다,,, 순간 여러가지 생각이 드는..😂
정말 고생 많으셨어요!!

앞으로도 같이 화이팅해봐요!🙌🙌

**Comment by euneunsol** [comment]
태영님! 
다른 팀이였지만 팀의 분위기 메이커란 소문을 들었습니다!
회고글에서도 분위기 메이커였단게 느껴지는 읽기 편하고 재미있는 글이였습니다!

하나만 피드백 드리자면..! 중간에 실행 화면 사진의 화질이 약간 아쉬웠습니다..
내용적인 부분에선 다 좋았던것 같습니다 :)

**Comment by ehddk** [comment]
태영님 회고록 잘 읽었습니다!
분위기 메이커이신 만큼 블로그에서도 재치가 살짝 느껴져서 술술 읽히네요. 프로젝트 개요 부분에 기간을 더 추가했으면 좋았을것 같다는 생각이 듭니당.
'저 높은 곳에서 다시 만나기를 바라며'라는 말씀이 확 와닿네요...저희도 열심히 배워서 능숙한 개발자가 되자구요😊👩🏻‍💻
